pkg/source/mtadapter/adapter.go

```diff
@@ -133,54 +133,11 @@ func (a *Adapter) Update(ctx context.Context, obj *v1beta1.KafkaSource) error {
 		return nil
 	}
 
-	saslUser, err := a.ResolveSecret(ctx, obj.Namespace, obj.Spec.Net.SASL.User.SecretKeyRef)
```

Comment:
should we also remove ResolveSecret defined in adapter.go?

Reply:
good catch! Thanks!